tools/run-app.js

```diff
@@ -243,7 +243,8 @@ _.extend(AppProcess.prototype, {
     // Call node
     var child_process = require('child_process');
     var child = child_process.spawn(nodePath, opts, {
-      env: self._computeEnvironment()
+      env: self._computeEnvironment(),
+      stdio: ['pipe', 'pipe', 'pipe', 'ipc']
```

Comment:
Probably only do this if watchForChanges is set.  ie, if `--once` is passed then we shouldn't create this extra fd.  (which might actually have an effect on whether or not the process terminates if the fd isn't disconnected?)